.github/dependabot.yml

```diff
@@ -1,5 +1,13 @@
 version: 2
 updates:
+- package-ecosystem: cargo
+  directory: "/"
+  schedule:
+    interval: weekly
```

Comment:
For git submodules we use daily interval, is it too frequent to use daily for cargo crates as well?

Reply:
We were planning to update the submodule updates to weekly for all rust-vmm components as otherwise there is too much noise. If there is a need to update it more frequently that weekly, we can still do that manually, but there is not a good way of silencing dependabot when you don't want so many PRs (except reducing the frequency).

Reply:
Makes sense, thanks :-)